bridges/ArtStationBridge.php

```diff
@@ -0,0 +1,84 @@
+<?php
+class ArtStationBridge extends BridgeAbstract {
+	const NAME = 'ArtStation';
+	const URI = 'https://www.artstation.com';
+	const DESCRIPTION = 'Fetches the latest ten artworks from a search query on ArtStation.';
+	const MAINTAINER = 'thefranke';
+	const CACHE_TIMEOUT = 3600; // 1h
+
+	const PARAMETERS = array(
+		'Search Query' => array(
+			'q' => array(
+				'name' => 'Search term',
+				'required' => true
+			)
+		)
+	);
+
+	public function getIcon() {
+		return 'https://www.artstation.com/assets/favicon-58653022bc38c1905ac7aa1b10bffa6b.ico';
+	}
+
+	public function getName() {
+		return self::NAME . ': ' . $this->getInput('q');
+	}
+
+	public function fetchSearch($searchQuery) {
```

Comment:
Any function that is not part of the API should be in private scope.

```suggestion
	private function fetchSearch($searchQuery) {
```